tests/shady-dynamic.html

```diff
@@ -1192,6 +1192,16 @@
     assert.equal(ShadyDOM.nativeTree.childNodes(fragment).length, 0, 'textContent setter on document fragment is incorrect');
   });
 
+  test('setting textContent to null or undefined should set it to empty string', function() {
```

Comment:
Let's split the `undefined` and `null` cases into two separate tests. That makes debugging easier when they fail 😄

Reply:
true that, done :+1: